-1

Tested at: e486fd1598e45967c608fb5db36fd23db6c70064


The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
https://github.com/cms-sw/cmssw/commit/e2f528f3334a7cdcc033799ce4e02c5b56cd1b00
You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-20970/23873/git-log-recent-commits
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-20970/23873/git-merge-result

You can see the results of the tests here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-20970/23873/summary.html

I found follow errors while testing this PR

**Failed tests:  Build**

* **Build**:

I found an error when building: 
 
 <pre>gmake[1]: Target 'PostBuild' not remade because of errors.
gmake[1]: Leaving directory '/build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_4_X_2017-10-19-1100'
config/SCRAM/GMake/Makefile.rules:2088: recipe for target 'src' failed
gmake: *** [src] Error 2
gmake: Target 'all' not remade because of errors.
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
</pre>


The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
https://github.com/cms-sw/cmssw/commit/e2f528f3334a7cdcc033799ce4e02c5b56cd1b00
You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-20970/23873/git-log-recent-commits
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-20970/23873/git-merge-result

--- a/DQM/TrackingMonitor/python/MonitorTrackInnerTrackMuons_cff.py
+++ b/DQM/TrackingMonitor/python/MonitorTrackInnerTrackMuons_cff.py
@@ -21,13 +21,6 @@ muonInnerTrack = SimMuon.MCTruth.MuonTrackProducer_cfi.muonTrackProducer.clone()
 muonInnerTrack.muonsTag = cms.InputTag("muonsPt10")
 muonInnerTrack.selectionTags = ('All',)
 muonInnerTrack.trackType = "innerTrack"
-muonInnerTrack.TkSizeBin = 10
-muonInnerTrack.TkSizeMax = 10.
-muonInnerTrack.phiErrMax = 0.001
-muonInnerTrack.etaErrMax = 0.001
-muonInnerTrack.PVBin =  40
-muonInnerTrack.PVMin =   0.5
-muonInnerTrack.PVMax = 120.5
 
 import DQM.TrackingMonitor.TrackingMonitor_cfi
 MonitorTrackMuonsInnerTrack = DQM.TrackingMonitor.TrackingMonitor_cfi.TrackMon.clone()
@@ -48,6 +41,13 @@ MonitorTrackMuonsInnerTrack.doDCAwrt000Plots = False
 MonitorTrackMuonsInnerTrack.doSIPPlots  = True
 MonitorTrackMuonsInnerTrack.doEffFromHitPatternVsPU = True
 MonitorTrackMuonsInnerTrack.doEffFromHitPatternVsBX = False
+MonitorTrackMuonsInnerTrack.TkSizeBin = 10
+MonitorTrackMuonsInnerTrack.TkSizeMax = 10.
+MonitorTrackMuonsInnerTrack.phiErrMax = 0.001
+MonitorTrackMuonsInnerTrack.etaErrMax = 0.001
+MonitorTrackMuonsInnerTrack.PVBin =  40
+MonitorTrackMuonsInnerTrack.PVMin =   0.5
+MonitorTrackMuonsInnerTrack.PVMax = 120.5
 
 #MonitorTrackINNMuons = cms.Sequence(muonInnerTrack+MonitorTrackMuonsInnerTrack)
 MonitorTrackINNMuons = cms.Sequence(cms.ignore(muonsPt10)+muonInnerTrack+MonitorTrackMuonsInnerTrack)

--- a/DQM/TrackingMonitor/python/TrackingMonitor_cfi.py
+++ b/DQM/TrackingMonitor/python/TrackingMonitor_cfi.py
@@ -391,10 +391,6 @@ TrackMon = cms.EDAnalyzer("TrackingMonitor",
     GoodPVtxMin = cms.double( 0.),
     GoodPVtxMax = cms.double(200.),
 
-    PVBin  = cms.int32 ( 150 ),
-    PVMin  = cms.double(   0.5),
-    PVMax  = cms.double( 150.5),
-
     LUMIBin  = cms.int32 ( 300 ),   # irrelevant
     LUMIMin  = cms.double(  200.),
     LUMIMax  = cms.double(20000.),
@@ -418,6 +414,10 @@ LongDCAMax = cms.double(8.0),
 
 # Overcoming the 255 arguments limit
 # TrackingRegion monitoring
+TrackMon.PVBin = cms.int32 ( 150 )
+TrackMon.PVMin = cms.double(   0.5)
+TrackMon.PVMax = cms.double( 150.5)
+
 TrackMon.RegionProducer = cms.InputTag("")
 TrackMon.RegionCandidates = cms.InputTag("")
 TrackMon.doRegionPlots = cms.bool(False)

